dashboard/app/models/user_level.rb

```diff
@@ -84,7 +84,12 @@ def navigator?
 
   def self.most_recent_driver(script, level, user)
     most_recent = find_by(script: script, level: level, user: user).try(:driver_user_levels).try(:last)
-    most_recent ? most_recent.user.name : nil
+    return nil unless most_recent
+
+    most_recent_user = most_recent.user
+    return 'Deleted user' unless most_recent_user
```

Comment:
can we internationalize this? It will [ultimately get passed](https://github.com/code-dot-org/code-dot-org/blob/4a13c1e/apps/src/StudioApp.js#L520) to [an i18n string](https://github.com/code-dot-org/code-dot-org/blob/8ed1268f2d616b50b46ee785acac95dafd435320/i18n/locales/es-ES/blockly-mooc/common.json#L220), and it'd be a bit weird to see `"Este rompecabezas fue completado mientras que se emparejaba con Deleted user"`